app/redux/redux-actions/adminActions.js

```diff
@@ -23,15 +23,124 @@ import {
   LOAD_SEARCHED_IMAGES,
   KEEP_LOADED_IMAGE_INFO,
   LOAD_MODAL_LIBRARY,
+  GET_ADMIN_MESSAGES,
+  GET_TEAM_MESSAGES,
+  UPDATE_HEAP,
+  LOAD_CC_ACTIONS,
 } from "../ReduxConstants";
 import { apiCall } from "../../utils/messenger";
 import { getTagCollectionsData } from "../../api/data";
 
+export const reduxFetchInitialContent = (auth) => (dispatch) => {
+  if (!auth) return;
+  const isSuperAdmin = auth && auth.is_super_admin;
+  Promise.all([
+    apiCall(
+      isSuperAdmin
+        ? "/communities.listForSuperAdmin"
+        : "/communities.listForCommunityAdmin"
+    ),
+    apiCall(
+      isSuperAdmin
+        ? "/actions.listForSuperAdmin"
+        : "/actions.listForCommunityAdmin"
+    ),
+    apiCall(
+      isSuperAdmin
+        ? "/events.listForSuperAdmin"
+        : "/events.listForCommunityAdmin"
+    ),
+    apiCall(
+      isSuperAdmin
+        ? "/messages.listForSuperAdmin"
```

Comment:
Right now, there is no messages.listForSuperAdmin;  we can add that but like most of these routes, listForCommunityAdmin switches on super admin (all communities) functionality when context says it is a super admin

Reply:
ooo okay great, even better. Will switch them.

Reply:
Thats good on the backend, but can be confusing on the frontend. I think mainly because of the naming.... I was a bit rattled when I saw that every route was saying `...forCommunityAdmin` 😅   was wondering how everything was working.. or if everything had been working like its supposed to even????? 🤣  Right now, I know because you told me, the next (developer) might not know, so it will probably help if we let things stay like this, so that even if it doesnt matter on the backend, things are still a bit clearer here....

That said, I will change the messages route thingy ... ASAP.